docs/README.md

```diff
@@ -4,7 +4,7 @@ To install required dependencies run the following:
 ``pip install sphinx sphinx-autobuild sphinx_rtd_theme``
 
 To start server, run following in metax-api directory:
-``sphinx-autobuild docs/source/ docs/build/``
+``sphinx-autobuild -t {envtag} docs/source/ docs/build/``, where {envtag} is one of local_development, test, stable or production. Envtag determines the target environment of the documentation.
```

Comment:
What does this envtag do?

Reply:
This tag controls the 'only' -directive in the documentation. This tag is also added to ansible playbook in ops repo https://github.com/CSCfi/metax-ops/pull/26/files.